inc/question.class.php

```diff
@@ -836,10 +836,17 @@ public function showForm($ID, $options = []) {
       if ($ldap_values === null) {
          $ldap_values = [];
       }
+      $current_entity = $_SESSION['glpiactive_entity'];
+      if ($current_entity == 0) {
+        $auth_ldap_condition = '';
+      } else {
+        $auth_ldap_condition = "glpi_authldaps.id in (select a.id from glpi_entities as e ,glpi_authldaps as a where a.id=e.authldaps_id and e.id=${current_entity})";
+      }
       Dropdown::show('AuthLDAP', [
          'name'      => 'ldap_auth',
+         'condition' => $auth_ldap_condition,
          'rand'      => $rand,
-         'value'     => (isset($ldap_values['ldap_auth'])) ? $ldap_values['ldap_auth'] : '',
+         'value'     => '',
```

Comment:
The field must show the value currently stored in DB. If one edit a question it will need to set again the LDAP to use. This might create headache to users.